docs/sources/flow/reference/components/prometheus.remote_write.md

```diff
@@ -354,3 +340,31 @@ information.
   remote storage.
 * `prometheus_remote_storage_exemplars_in_total` (counter): Exemplars read into
   remote storage.
+
+## Example
+
```

Comment:
Later might want to add an introduction to each example to say what's going on. The comments basically do the trick but it's better to have something in the text.